inc/features/blocks/personalization/components/variant-toolbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const { IconButton } = wp.components;
+const { __ } = wp.i18n;
+
+const VariantToolbar = props => {
+	const {
+		canRemove,
+		isFallback,
+		onCopy,
+		onRemove,
+	} = props;
+
+	return (
+		<div className="altis-experience-block-header__toolbar">
+			<IconButton
+				icon='migrate'
+				title={ __( 'Copy variant', 'altis-experiments' ) }
+				onClick={ onCopy }
+			>
+				{ __( 'Copy', 'altis-experiments' ) }
+			</IconButton>
+			{ ! isFallback && (
+				<IconButton
+					icon='trash'
+					title={ __( 'Remove variant', 'altis-experiments' ) }
+					disabled={ ! canRemove }
```

Comment:
Sort and double-quote:

```suggestion
					disabled={ ! canRemove }
					icon="trash"
					title={ __( 'Remove variant', 'altis-experiments' ) }
```